src/vm/methodtablebuilder.cpp

```diff
@@ -8475,7 +8475,7 @@ void MethodTableBuilder::SystemVAmd64CheckForPassStructInRegister()
     // Iterate through the fields and make sure they meet requirements to pass in registers
     SystemVStructRegisterPassingHelper helper((unsigned int)totalStructSize);
 
-    if (GetHalfBakedMethodTable()->ClassifyEightBytes(&helper, 0, 0))
+    if (GetHalfBakedMethodTable()->ClassifyEightBytes(&helper, 0, 0, false))
```

Comment:
Nit: it would be nice to use a named constant in these two calls instead of `false`/`true`. Something like:

```
const bool UseNativeLayout = false;
if (GetHalfBakedMethodTable()->ClassifyEightBytes(&helper, 0, 0, false))
```

and

```
const bool UseNativeLayout = true;
if (GetHalfBakedMethodTable()->ClassifyEightBytes(&helper, 0, 0, true))
```